### ISSUE TITLE
Add the swipe on the emoji screen

### ISSUE THREAD

**Issue by gab12** [florisboard/florisboard#647]
Hello,

In the screen emoji icon. there are sub categories of emoji (smiley, guys, flowers ....)
If you want to change of categories, you have to click on the icon.
Personally, I change categories very often.

It would be nice to be able to change category, just on a left or rigth swipe on the emoji area(in addition to a click on the iconà.
It would make the entry much easier. In any case, the swipe is natural on this screen and it seems missing.
Thank you and well done for your work.

![Screenshot](https://user-images.githubusercontent.com/6320898/114572103-ba3ba380-9c66-11eb-96dc-0c9d50199a66.jpg)

**Comment by Bappamaiti** [comment]
Nice suggestion. I agreed.

**Comment by patrickgold** [comment]
Thanks for your suggestion! Will add this one to my emoji rework todo list, which I will work on for the v0.5.0 milestone (current milestone is v0.4.0).

**Comment by gab12** [comment]
I just wanted to post a small update, as it’s been three years since I proposed this feature and I’m still eagerly awaiting it. With version 5 in progress (and even version 6 on the horizon), I just wanted to share that this request is still relevant and would be greatly appreciated if it were to come to life! 🙂

Of course, please take all the time you need, and thank you for the amazing work you’re doing on this open-source project!

**Comment by Glitchy-Tozier** [comment]
I agree that this would be a great feature, but beforehand we should probably fix #2638 to not prompt tons of additional bug-reports.

**Comment by lm41** [comment]
I've finally figured out a way how to do that 🎉. If you have time you can test the debug build [1]. 

> [!NOTE]
> Debug builds can be quite laggy. On my device (p7p) was the performance not the best, but it ran very smooth in the emulator.  


[1] https://github.com/florisboard/florisboard/actions/runs/16540950370/artifacts/3621380036
